themes/vuetober/src/views/Feedback.vue

```diff
@@ -0,0 +1,184 @@
+            :src="require(`@/assets/img/feedplus.png`)"
+            alt="+"
+          >
+          <div class="ml-5 mb-3 w-100">
+            <h6 class="text-muted font-weight-bold">
+              SPŠE Hálova
+            </h6>
+            <h1 class="font-weight-bold">
+              Pridať feedback
+            </h1>
+            <p class="text-muted">
+              Pridajte si učiteľa a predmet, ktorý vyučuje
+            </p>
+            <b-form
+              inline
+              class="mt-4"
+            >
+              <b-form-select
+                class="mr-2 form1"
+                v-model="feedback.teacherId"
+              >
+                <b-form-select-option :value="0">
+                  Meno učiteľa
+                </b-form-select-option>
+                <b-form-select-option
+                  v-for="teacher in teachers"
+                  :key="teacher.id"
+                  :value="teacher.id"
+                >
+                  {{ teacher.name }}  {{ teacher.surname }}
+                </b-form-select-option>
+              </b-form-select>
+              <b-form-select
+                class="mr-2 form1"
+                v-model="feedback.subjectId"
+                :disabled="feedback.teacherId === 0"
+              >
+                <b-form-select-option :value="0">
+                  Predmet
+                </b-form-select-option>
+                <b-form-select-option
+                  v-for="subject in subjects[feedback.teacherId-1]"
+                  :key="subject.id"
+                  :value="subject.id"
+                >
+                  {{ subject.subject_name }}
+                </b-form-select-option>
+              </b-form-select>
+              <b-button
+                variant="danger"
+                class="px-3 py-2"
+                @click.prevent="createFeedback"
+              >
+                Odoslať feedback
+              </b-button>
+            </b-form>
+          </div>
+        </div>
+        <hr>
+
+        <b-form>
+          <b-form-textarea
+            id="textarea-rows"
+            placeholder="Tall textarea"
+            rows="8"
+            class="textfield"
+            v-model="feedback.feedback"
+          />
+        </b-form>
+      </b-col>
+    </b-row>
+  </b-container>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+
+export default {
+  name: 'Feedback',
+  data() {
+    return {
+      feedback: {
+        feedback: '',
+        teacherId: 0,
+        subjectId: 0,
+      },
+    };
+  },
+  created() {
+    this.$store.dispatch('fetchTeachers');
+  },
+  methods: {
+    createFeedback() {
+      const { teacherId, feedback } = this.feedback;
+      this.$store.dispatch('createFeedback', { teacherId, feedback }).then(() => {
+        this.$router.push('teachers');
+      });
+    },
+  },
+  computed: {
+    ...mapGetters([
+      'getTeachers',
+      'getSelectedTeacher',
+    ]),
+    teachers() {
+      return this.getTeachers('');
```

Comment:
Toto treba znovu definivoat? Nestaci len zavolat v template `getTeachers()`?

Reply:
Hej, da sa to skratit, len kvoli sortovaniu v state potrebuje aspon empty string ako parameter.